src/System.Management.Automation/help/MUIFileSearcher.cs

```diff
@@ -133,13 +133,11 @@ private string[] GetFiles(string path, string pattern)
                     result.Add(filePath);
                     break;
                 }
-                // If the input is pattern instead of string, we need to use Regex expression.
-                if (pattern.Contains("*") || pattern.Contains("?"))
+
+                string fileName = Path.GetFileName(filePath);
```

Comment:
`fileName` isn't used when `wildcardPattern` is null, so we should avoid the call (which copies and allocates memory) unless we'll actually use the value.